pkg/config/config.go

```diff
@@ -796,10 +796,19 @@ func (c *Config) ToFile(path string) error {
 	if err != nil {
 		return err
 	}
-
 	return os.WriteFile(path, b, 0o644)
 }
 
+// LogConfig pretty prints current config
+func (c *Config) LogConfig() error {
+	tomlConfig, err := c.ToBytes()
+	if err != nil {
+		return err
+	}
+	logrus.Infof("Current runtime configuration: \n%s", tomlConfig)
```

Comment:
not to bikeshed, but `runtime` is often overloaded, maybe just `Current CRI-O configuration:`?